ui/app/page.tsx

```
@@ -0,0 +1,195 @@
+          <Card className="border-border">
+            <CardHeader>
+              <CardTitle className="flex items-center gap-2">
+                <Sparkles className="h-5 w-5 text-primary" />
+                Key Features
+              </CardTitle>
+            </CardHeader>
+            <CardContent>
+              <div className="space-y-4 text-sm">
+                {keyFeatures.map((feature, index) => (
+                  <div key={index} className="flex items-center space-x-3">
+                    <div className="w-1.5 h-1.5 bg-primary rounded-full"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)